code/__DEFINES/misc.dm

```diff
@@ -415,6 +415,7 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 #define SCAR_LOC_FILE "wounds/scar_loc.json"
 #define EXODRONE_FILE "exodrone.json"
 #define CLOWN_NONSENSE_FILE "clown_nonsense.json"
+#define CULT_SHUTTLE_CURSE "cult_shuttle_curse.json"
```

Comment:
we should really eventually move either all the string path defines into their own file or files that make the most sense for them in the future

not something you have to do btw just food for thought